src/test/routes/auth.spec.js

```diff
@@ -26,6 +26,28 @@ describe('AUTH', () => {
         });
     });
 
+    it('should not #create a user with wrong password', (done) => {
```

Comment:
Nice work bro, I think the description, for this test is not descriptive enough.

Reply:
Okay, i will update it. Thank you